staff/pedrovalverde/css-layout-adaptive/styles/main.css

```diff
@@ -0,0 +1,3 @@
+
+@import url("mobile.css") (max-width: 768px);
+@import url("desktop.css") (min-width: 768px);
```

Comment:
there are common properties in both versions (mobile and desktop), and should be in the main, so we shortened the length of both CSS files.